respa_admin/static_src/js/periods.js

```diff
@@ -322,11 +322,10 @@ function restoreDaysMgmtFormValues(periodItem) {
 * in the corresponding period.
 * */
 function updateTotalDays($periodItem) {
-  let amountOfDays = $periodItem.find('#period-days-list').children().length;
-  let amountOfOriginalDays = $periodItem.find('#period-days-list').children('.weekday-row.original-day').length;
-  let daysMgmtForm = $periodItem.find('#days-management-form');
+  const daysMgmtForm = $periodItem.find('#days-management-form');
+
+  const amountOfDays = $periodItem.find('#period-days-list').children().length;
   daysMgmtForm.find("[id$='-TOTAL_FORMS']").val(amountOfDays);
-  daysMgmtForm.find("[id$='-INITIAL_FORMS']").val(amountOfOriginalDays);
```

Comment:
This was the issue with the original bug: `INITIAL_FORMS` is set to the number of forms, which breaks validation on POST.

This is because Django formsets check this value for the pre-existing forms, which should always be zero for new periods (or copies of existing periods).